src/main/app/api/CosApi.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosInstance } from 'axios';
+import { LoggerInstance } from 'winston';
+
+export class CosApi {
+  constructor(private readonly axios: AxiosInstance, private readonly logger: LoggerInstance) {}
+
+  //TODO change return type when backend has been implemented.
+  public getCase(): Promise<Record<string, unknown>> {
+    return this.axios
+      .get('/retrieve-case')
+      .then(results => results.data)
+      .catch(err => {
+        this.logger.error(err);
+        return false;
+      });
+  }
```

Comment:
Looks like this method needs a test to get it up [above the 60% coverage from sonarcloud](https://sonarcloud.io/component_measures?id=nfdiv-frontend&pullRequest=174&metric=new_coverage&view=list)